labs/vstsextend/sonarcloud/readme.md

```diff
@@ -14,9 +15,9 @@ Technical debt is the set of problems in a development effort that make forward
 [SonarCloud](https://about.sonarcloud.io/){:target="_blank"} is the code quality cloud service provided by SonarSource.
```

Comment:
Not changed in this commit but might be worth checking with Fabrice whether to change this to `the code quality and security service...`